security_test.go

```diff
@@ -236,17 +236,13 @@ var _ = Describe("Security", func() {
 		})
 
 		Context("when running a command as a non-root user", func() {
-			JustBeforeEach(func() {
-				createUser(container, "alice")
-			})
-
 			It("executes with correct uid, gid, and supplementary gids", func() {
 				stdout := runForStdout(container, garden.ProcessSpec{
 					User: "alice",
 					Path: "/bin/sh",
 					Args: []string{"-c", "id -u; id -g; id -G"},
 				})
-				Expect(stdout).To(gbytes.Say("1001\n1001\n1001\n"))
+				Expect(stdout).To(gbytes.Say("1001\n1010\n1010 1011\n"))
```

Comment:
Is there a missing newline here?  The string looks a bit odd.

Reply:
No, alice is just part of two groups. Context: https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/main/garden-runc-release/dockerfiles/garden-rootfs/Dockerfile#L81-L82